holmes/plugins/toolsets/mcp/toolset_mcp.py

```diff
@@ -290,5 +339,6 @@ def get_example_config(self) -> Dict[str, Any]:
             url=AnyUrl("http://example.com:8000/mcp/messages"),
             mode=MCPMode.STREAMABLE_HTTP,
             headers={"Authorization": "Bearer YOUR_TOKEN"},
+            context_fields=["tool_name", "tool_number", "request_context"],
```

Comment:
In holmesgpt, we can leverage BaseModel to init a struct, so that we don't need _serialize_context.
I think we can remove the duplicated field that are defined already in ToolInvokeContext, basically tool_name and tool_number, which are for internal use for CLI experience.

Reply:
removed the fields.